packages/walletconnect/src/utils.ts

```diff
@@ -1,20 +1,22 @@
 /**
  * @param rpcMap - Map of chainIds to rpc url(s).
- * @param timeout - How long to wait before a call is considered failed, in ms.
+ * @param timeout - Timeout, in milliseconds, after which to treat network calls to urls as failed when selecting
+ * online urls.
  */
-export async function getRpcBestUrlMap(rpcMap: Record<string, string | string[]>, timeout: number) {
+export async function getBestUrlMap(rpcMap: Record<string, string | string[]>, timeout: number) {
   return Object.fromEntries(
     await Promise.all(
       Object.entries(rpcMap).map(
-        async ([chainId, map]): Promise<[string, string]> => [`${chainId}`, await getBestUrl(map, timeout)]
+        async ([chainId, map]): Promise<[string, string]> => [chainId, await getBestUrl(map, timeout)]
       )
     )
   )
 }
 
 /**
  * @param urls - An array of URLs to try to connect to.
- * @param timeout - @see getRpcBestUrlMap#timeout
```

Comment:
nit: I liked that you referred to getRpcBestUrlMap#timeout

Reply:
Yeah, me too, but the link wasn't working. Now that I took another pass, `{@link ...}` works like a charm!